nest-tax-backend/.env.example

```diff
@@ -15,6 +15,10 @@ PAYGATE_MERCHANT_NUMBER=
 PAYGATE_CURRENCY=
 PAYGATE_PAYMENT_REDIRECT_URL=
 PAYGATE_PASSPHRASE=
+PAYGATE_REDIRECT_URL=
+PAYGATE_AFTER_PAYMENT_REDIRECT_FRONTEND=
+PAYGATE_SIGN_CERT=
+PAYGATE_KEY=
```

Comment:
Discuss with @JHvolka how to make them reasonably populated, as he did some work on this example yesterday. So please resolve the merge conflict and populate these according to his guidance.